src/Classes/Api.php

```diff
@@ -0,0 +1,61 @@
+<?php
+namespace ListingsApp\Classes;
+/**
+ * send api request to receive listings data
+ */
```

Comment:
Don't need a doc block for a class. Unless we do, in which case please let me know so we can doc block ours!